docs/workflows/genomic_characterization/theiacov.md

```diff
@@ -193,6 +193,30 @@ All TheiaCoV Workflows (not TheiaCoV_FASTA_Batch)
 | gene_coverage | **sc2_s_gene_stop** | Int | End/Last nucleotide position of the SARS-CoV-2 Spike gene | 25384 | Optional | CL, ONT, PE, SE | MPXV, sars-cov-2 |
 | ivar_consensus | **read2** | File | Internal component, do not modify | | Do not modify, Optional | SE | HIV, MPXV, WNV, rsv_a, rsv_b, sars-cov-2 |
```

Comment:
this is super nitpicky here: can you put these two older ones in alphabetical order with your additions?? thanks!!!